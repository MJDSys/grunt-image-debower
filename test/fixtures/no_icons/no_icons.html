<ul>
<li>Test page</li>
<li>No icons</li>
</ul>
