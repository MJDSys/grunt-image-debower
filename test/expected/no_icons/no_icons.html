<ul>
<li>Test page
<li>No icons
</ul>
